cadc-util/src/main/java/ca/nrc/cadc/util/RsaSignatureGenerator.java

```diff
@@ -150,86 +148,94 @@ public RsaSignatureGenerator(File keyFile) {
         super(keyFile, true);
         initPrivateKey(keyFile);
     }
+
```

Comment:
I think RsaSignatureGenerator needs a method like
```
public static KeyPair generateKeyPair(int len)
```

It's essentially the first half of the current generateKeyPair(File, File, int) method. The code in raven and vault can generate in-memory keys when necessary; cavern needs the existing File based method. I'm assuming byte correct `byte[]` is available via KeyPair methods (looks like it).